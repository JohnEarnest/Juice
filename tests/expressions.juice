// tests of various operator combos and precedence

print(0.4 + 0.3);      println();
print(0.4 + 1);        println();
print(3 + 2 * 3);      println();
print((3+2) * 3);      println();
print(3 < 4);          println();
print(3 > 4 && 3 < 5); println();