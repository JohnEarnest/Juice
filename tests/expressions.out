0.70000005
1.4
9
15
true
false
